src/sentry/grouping/strategies/message.py

```diff
@@ -151,40 +154,78 @@
 
 _parameterization_regex = re.compile(_parameterization_regex_str)
 
+# UniqID logic
+encoding = tiktoken.get_encoding("cl100k_base")
```

Comment:
this should not be at the module scope as it significantly impacts import time (module scope should have no side-effects and nothing expensive as imports in python are just executing the module scope)

you can usually hide this behind lazy initialization using `lru_cache`